src/events.js

```diff
@@ -13,16 +13,17 @@ export const paneConnected = "l:pane:connected";
 /**
  * @param {HTMLElement} el
  * @param {Evented} evented
+ * @param bubble determines whether events should bubble (defaults to true)
  */
-export const connectLeafletEvents = (el, evented) => {
+export const connectLeafletEvents = (el, evented, bubble = true) => {
   if (el.hasAttribute("on")) {
     const on = el.getAttribute("on");
     if (on !== null) {
       on.split(/\s+/).forEach((eventName) => {
         if (evented !== null) {
           evented.on(eventName, (e) => {
             el.dispatchEvent(
-              new CustomEvent(eventName, { bubbles: true, detail: e })
+              new CustomEvent(eventName, { bubbles: bubble, detail: e })
```

Comment:
This can then be `... { bubbles, detail: e })`

Reply:
same as above